Cargo.toml

```diff
@@ -99,27 +102,34 @@ _sqlite = []
 _bundled = ["duckdb/bundled", "rusqlite/bundled"]
 
 [dev-dependencies]
-tokio = { version = "1.19", features = ["rt-multi-thread", "net", "macros", "time"]}
+tokio = { version = "1.19", features = [
+  "rt-multi-thread",
+  "net",
+  "macros",
+  "time",
+] }
 rustls-pki-types = { version = "1.10" }
 rusqlite = { version = "0.37.0", features = ["column_decltype"] }
 ## for duckdb example
-duckdb = { version = "1.0.0" }
+duckdb = { version = "=1.3.2" }
```

Comment:
Instead of pinning to 1.3.2, it better to use something like `1.3, <1.4` (i don't remember the exact semver syntax, but it looks like that)